src/simulator_core/entities/assets/asset_abstract.py

```diff
@@ -16,24 +16,17 @@
 """Abstract class for asset."""
 
 from abc import ABC, abstractmethod
-from typing import Dict, List, TypeVar
+from typing import Dict, List
 
 from pandapipes import pandapipesNet
 from pandas import DataFrame
-
-Junction = TypeVar("Junction")
-EsdlAssetObject = TypeVar("EsdlAssetObject")
+from simulator_core.entities.assets.esdl_asset_object import EsdlAssetObject
+from simulator_core.entities.assets.junction import Junction
 
 
 class AssetAbstract(ABC):
     """Abstract class for Asset."""
 
```

Comment:
Waarom heb je dit moeten aanpassen? Sebastiaan had aangegeven dat dit de "beste/standaard" manier was. Nu moet je overal weer self.from_junction: Junction | None toevoegen en daarnaast ook nog een docstring? Kunnen we dit een keer bespreken of in een issue plaatsen?

Reply:
Aangepast omdat mypy hierover viel. dit kunnen we meenmeen in issue #58